xbmc/music/MusicDatabase.cpp

```diff
@@ -4069,11 +4080,17 @@ void CMusicDatabase::UpdateTables(int version)
   {
     m_pDS->exec("ALTER TABLE song ADD mood text\n");
   }
+  if (version < 53)
+  {
+    m_pDS->exec("ALTER TABLE song ADD dateAdded text\n");
```

Comment:
Your right, what about the other migrations that have these? Should I change them too?